### PR TITLE
Add know issue on CSIMigrationvSphere for 1.22

### PR DESCRIPTION
**Pull request by pmmalinov01**
Signed-off-by: Pavel Malinov <pmmalinov01@gmail.com>

#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:
Notifies users to now migrate to the newest release based on a recently discovered problem

#### Which issue(s) this PR fixes:
FIxes https://github.com/kubernetes/kubernetes/issues/101728

#### Special notes for your reviewer:
/cc @sladyn98 @cici37  @Damans227 @simcard0000 @savitharaghunathan @xing-yang

### CODE REVIEW

File: releases/release-1.22/release-notes/release-notes-draft.md [hunk 1]
```diff
@@ -66,6 +66,13 @@ The API used to create [Ephemeral Containers](https://kubernetes.io/docs/concept
 For stable features, the `kubectl` tool follows the Kubernetes [version skew policy](https://kubernetes.io/releases/version-skew-policy/);
 however, kubectl v1.21 and older do not support the new API for ephemeral containers.
 Users who create ephemeral containers using `kubectl debug` should note that kubectl version 1.22 will attempt to fall back to the old API; older versions of kubectl will not work with cluster versions of 1.22 or later. Please update kubectl to 1.22 if you wish to use `kubectl debug` with a mix of cluster versions.
+
+## Know Issues
```
Comment (xing-yang) [inline releases/release-1.22/release-notes/release-notes-draft.md:70]: Know -> Known

File: releases/release-1.22/release-notes/release-notes-draft.md [hunk 2]
```diff
@@ -66,6 +66,13 @@ The API used to create [Ephemeral Containers](https://kubernetes.io/docs/concept
...
+
+### `CSIMigrationvSphere` feature gate has not migrated to new CRD APIs
+
+If CSIMigrationvSphere feature gate is enabled, user should not upgrade to Kubernetes v1.22. vSphere CSI Driver does not support Kubernetes v1.22 yet because it uses v1beta1 CRD APIs. Support for v1.22 will be added at a later release. Check the following document for supported Kubernetes releases for a given [vSphere CSI Driver version](https://vsphere-csi-driver.sigs.k8s.io/compatiblity_matrix.html#compatibility-matrix-for-vsphere-csi-driver)
```
Comment (xing-yang) [inline releases/release-1.22/release-notes/release-notes-draft.md:74]: Add a period at the end.

**Review by xing-yang — COMMENTED** [review]
A couple of nits.  Otherwise looks good.

**Review by savitharaghunathan — APPROVED** [review]
/lgtm 
/approve

@pmmalinov01 feel free to cancel hold when you are ready

**Pull request merged**